packages/jupyter-ai-magics/jupyter_ai_magics/magics.py

```diff
@@ -169,7 +169,8 @@ def _ai_inline_list_models_for_provider(self, provider_id, Provider):
                 return Provider.help
 
         for model_id in Provider.models:
-            output += f", `{provider_id}:{model_id}`"
+            # Wrap the name in an HTML tag to make all hyphens nonbreaking
+            output += f", <span style='white-space: nowrap; font-family: monospace;'>{provider_id}:{model_id}</span>"
```

Comment:
I am not suggesting to change it if you already considered it and decided against it, but just wanted to make sure you are aware that we could do:

```suggestion
            output += f", <span style='white-space: nowrap;'>`{provider_id}:{model_id}`</span>"
```

Another alternative would be to just use a list.

Reply:
That was what I tried first. This causes `{provider_id}:{model_id}` to be wrapped with a `<code>` tag, whose `white-space` style overrides the explicit `nowrap`.